src/Sprites/spriteManager.ts

```diff
@@ -401,6 +410,68 @@ export class SpriteManager implements ISpriteManager {
         return null;
     }
 
+    /**
+     * Intersects the sprites with a ray
+     * @param ray defines the ray to intersect with
+     * @param camera defines the current active camera
+     * @param predicate defines a predicate used to select candidate sprites
+     * @returns null if no hit or a PickingInfo array
+     */
+    public multiIntersects(ray: Ray, camera: Camera, predicate?: (sprite: Sprite) => boolean): Nullable<PickingInfo[]> {
+        var count = Math.min(this._capacity, this.sprites.length);
+        var min = Vector3.Zero();

```

Comment:
I would recommend to use min = TmpVectors.Vector3[0].copyFromFloats(0, 0, 0) and max =  TmpVectors.Vector3[1].copyFromFloats(0, 0, 0) to avoid instantiating vectors

Reply:
Just copied and pasted from existing Intersects method so will change those as well. (Grandchildren arriving back soon so if I don't get the changes done in the nest few mins they will have to wait until next week)

Reply:
done

Reply:
No problem man.Grandchidlren > all